reflex/components/radix/themes/components/text_field.py

```diff
@@ -68,6 +68,8 @@ class TextFieldRoot(elements.Div, RadixThemesComponent):
     # Specifies the type of input
     type: Var[str]
 
+    step: Var[int]
```

Comment:
If this is being included for documentation purposes, please add a preceeding comment, which will be used to generate the docs for this component.